<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans CJK TC Bold</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RS232 Port Connection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLineEdit" name="msg_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="msg_textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="send_msg_pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/prefix1/icon/send.png</normaloff>:/new/prefix1/icon/send.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/prefix1/icon/clear.png</normaloff>:/new/prefix1/icon/clear.png</iconset>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="port_status_label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>COM Prot Status：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>COM Prot Name：</string>
      </property>
     </widget>
     <widget class="QLabel" name="port_name_label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QPushButton" name="open_com_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Open COM</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/prefix1/icon/connect.png</normaloff>:/new/prefix1/icon/connect.png</iconset>
      </property>
     </widget>
     <widget class="QComboBox" name="port_name_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Port Name：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="port_baud_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Baudrate：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Parity：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="port_parity_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EVEN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MARK</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="port_dataBit_comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Bit：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Bit：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="port_stopBit_comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow Control：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="port_flowControl_comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SOFTWARE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HARDWARE</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="close_com_pushButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close COM</string>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/new/prefix1/icon/disconnect.png</normaloff>:/new/prefix1/icon/disconnect.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTransmit_file"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTransmit_file">
   <property name="text">
    <string>Transmit File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clear_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>msg_textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>msg_lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>send_msg_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
